core/shared/src/main/scala/org/http4s/ProductIdOrComment.scala

```diff
@@ -23,10 +23,15 @@ import org.http4s.util.Writer
 
 sealed trait ProductIdOrComment extends Renderable
 object ProductIdOrComment {
-  private[http4s] val serverAgentParser: P[(ProductId, List[ProductIdOrComment])] = {
+  @deprecated("Use serverAgentParser(Int) instead", "0.22.15")
```

Comment:
Should we fix the 'deprecation since' value?

Reply:
The non-linearity of releases is hard.  We've gone both ways.  0.23.17 is probably more useful on this branch, yes.